Libplanet.Store.Remote.Tests/Helpers/LoggerHelper.cs

```diff
@@ -0,0 +1,22 @@
+using Microsoft.Extensions.Logging;
```

Comment:
Any reason why we are using `Microsoft.Extensions.Logging` instead of `Serilog`?

Reply:
No. Since `Microsoft.Extensions.Logging` is provided by default without special Dependency injection, it was used for no particular reason during development. As you said, it would be better to change to Serilog.

Reply:
https://stackoverflow.com/questions/61544047/use-serilog-with-microsoft-extensions-logging-ilogger Ok, I found some information. use `Microsoft.Extensions.Logging` is ok even if the user wants to use `Serilog`.

Reply:
Yes, I understand it's "doable", but the link above is about how to integrate `Serilog` to existing large codebase using `Microsoft.Extensions.Logging`. Our whole stack is based on `Serilog` and unless there is a specific reason to introduce using `Microsoft.Extensions.Logging`, this just increases the overall maintenance cost, which I don't see it being justified. 😐

Reply:
AFAIK, `Microsoft.Extensions.Logging` is a dotnet-standard logging system. So in my opinion, the user can use only the serilog is not a proper way especially if it is a library.

What's your opinion about this?

Reply:
> AFAIK, Microsoft.Extensions.Logging is a dotnet-standard logging system. So in my opinion, the user can use only the serilog is not a proper way especially if it is a library.

Yes, I understand your sentiment. If we are to choose, using `Microsoft.Extensions.Logging` could be an option, but I'd say this requires a larger consensus among the engineers working on this project. If we do decide to move away from `Serilog` and we set it as a goal, I would have less qualms about this change. If we are just introducing it, especially in a "subproject" of [libplanet](libplanet.io), this becomes an unmanaged technical debt.

Reply:
Ok, I understand. I'll amend your request.

Reply:
For Asp users I think using Serilog can be inconvenient. I personally think it's better to create a Libplanet.ILogger and let the user decide which logger to use. 😀

Reply:
https://github.com/planetarium/libplanet/pull/3688/commits/45d8987d803e10488d371e376804d698d6dbbdb6 amended. @greymistcube